unit_test/Makefile

```diff
@@ -113,6 +113,7 @@ ifeq ($(KOKKOSKERNELS_INTERNAL_TEST_OPENMP), 1)
   OBJ_OPENMP += Test_OpenMP_Blas2_gemv.o
   OBJ_OPENMP += Test_OpenMP_Blas2_team_gemv.o
   OBJ_OPENMP += Test_OpenMP_Blas3_gemm.o
+  OBJ_OPENMP += Test_OpenMP_Blas3_trsm.o
```

Comment:
I think these need to get added to the CMake, too?

Reply:
@jjwilke they do not need to get added since I see the ```unit_test/CMakeLists.txt``` does something like
```APPEND_GLOB(OPENMP_BLAS_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/openmp/Test_OpenMP_Blas*.cpp)```

Reply:
Oops. My bad : )